src/routes/news/containers/LocalNewsPage.js

```diff
@@ -0,0 +1,85 @@
+// @flow
+
+import * as React from 'react'
+import { connect } from 'react-redux'
+import compose from 'lodash/fp/compose'
+import { LocalNewsModel, CityModel } from '@integreat-app/integreat-api-client'
+import type { TFunction } from 'react-i18next'
+import { withTranslation } from 'react-i18next'
+import type { StateType } from '../../../modules/app/StateType'
+import NewsElement from '../components/NewsElement'
+import LocalNewsList from '../components/LocalNewsList'
+import NewsTabs from '../components/NewsTabs'
+import { LOCAL_NEWS } from '../constants'
+import LoadingSpinner from '../../../modules/common/components/LoadingSpinner'
+import ContentNotFoundError from '../../../modules/common/errors/ContentNotFoundError'
+import FailureSwitcher from '../../../modules/common/components/FailureSwitcher'
+
+type PropsType = {|
+  localNews: Array<LocalNewsModel>,
+  city: string,
+  cities: Array<CityModel>,
+  areCitiesFetching: boolean,
+  language: string,
+  t: TFunction,
+  path: string
+|}
+
+export class LocalNewsPage extends React.Component<PropsType> {
+  renderLocalNewsElement = (language: string) => (localNewsItem: LocalNewsModel, city: string) => {
+    const { id, title, message, timestamp } = localNewsItem
+    return <NewsElement
+      id={id}
+      title={title}
+      content={message}
+      timestamp={timestamp}
+      key={id}
+      path={this.props.path}
+      t={this.props.t}
+      language={language}
+      type={LOCAL_NEWS}
+    />
+  }
+
+  render () {
+    const { localNews, city, cities, areCitiesFetching, path, language, t } = this.props
+
+    if (areCitiesFetching) {
+      return <LoadingSpinner />
+    }
+
+    const currentCity: CityModel = cities && cities.find(cityElement => cityElement.code === city)
+
+    if (!currentCity.pushNotificationsEnabled) {
+      const type = currentCity.tunewsEnabled ? 'tunewsItem' : 'category'
```

Comment:
shouldn't it be?
```suggestion
      const type = currentCity.tunewsEnabled ? 'tunewsItem' : 'localNewsItem'
```

Reply:
No, it shouldn't. In fact, imo it should even always be `category`. If a city disables the feature, the user shouldn't notice that the feature exists at all and rather be displayed a general not found error, which is in our case a category not found.

Reply:
This is meant to redirect the user to `category` route if both local news "pushNotificationsEnabled" and tunews were disabled.